camel/toolkits/hybrid_browser_toolkit/ts/websocket-server.js

```
@@ -234,10 +237,16 @@ class WebSocketBrowserServer {
             console.error('Error closing browser:', error);
           }
         }
+        // Return response immediately
+        const shutdownResponse = { message: 'Server shutting down' };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)